flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -409,7 +413,7 @@ public void testKeyedMapStateSerializerReconfiguration() throws Exception {
                         new TestType.ReconfigurationRequiringTestTypeSerializer()));
     }
 
-    @Test
+    @TestTemplate
     public void testKeyedMapStateRegistrationFailsIfNewStateSerializerIsIncompatible() {
```

Comment:
```suggestion
    void testKeyedMapStateRegistrationFailsIfNewStateSerializerIsIncompatible() {
```
could be package private